server/server.go

```diff
@@ -15,9 +16,11 @@ import (
 	"github.com/opentracing/opentracing-go"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	node_https "github.com/prometheus/node_exporter/https"
```

Comment:
Given the caution expressed in https://github.com/prometheus/node_exporter/pull/1677 I think there need to be some comments in the PR description and in the code noting that this code is awaiting security review.